packages/react-sdk/src/components/BoardBar/pdf/utils.ts

```diff
@@ -40,17 +40,74 @@ export function image(element: ImageElement, base64content: string): Content {
   };
 }
 
+function preprocessSvg(element: ImageElement, svg: string): string {
```

Comment:
Canvas _requires_ there to be a width and height on the svg element. Otherwise it wont draw it.